scene/gui/control.cpp

```diff
@@ -3029,18 +3025,6 @@ void Control::_notification(int p_notification) {
 			data.parent_window = Object::cast_to<Window>(get_parent());
 			data.is_rtl_dirty = true;
 
-			if (data.theme.is_null()) {
```

Comment:
I'm a bit concerned that this code is executed not just when entering the tree, but also when the node is set as top level. But that doesn't seem to have any effect on the theme, so should be fine.